### PR TITLE
Allow to add Models/Sitemapables and PHP8 only

### PR DESCRIPTION
**Pull request by Gummibeer**
fixes #367

This change shouldn't be breaking. The changes to the `Sitemap::add()` method allow the user to return a string (URL), an instance of an `Url` tag or even an array of these and all are added to the sitemap.

### CODE REVIEW

File: src/SitemapServiceProvider.php [hunk 1]
```diff
@@ -7,27 +7,29 @@
 
 class SitemapServiceProvider extends ServiceProvider
```
Comment (freekmurze) [inline src/SitemapServiceProvider.php:8]: We could use the `PackageServiceProvider` from spatie/laravel-package-tools to clean this up

Comment (Gummibeer) [reply src/SitemapServiceProvider.php]: https://github.com/spatie/laravel-sitemap/pull/368/commits/6aafa86724d4a3d7bcb719617040b03968154af2

**Comment by Gummibeer** [comment]
The view namespace has changed because of `spatie/laravel-package-tools` removing the `laravel-` prefix. So this is now a breaking change as custom views won't be applied now anymore.

**Comment by patinthehat** [comment]
@Gummibeer I just finished implementing `laravel-package-tools` for this package - but you beat me to it! Nice! :+1:

**Comment by Gummibeer** [comment]
@patinthehat Oh sorry for this duplication and miscommunication. 😔
I primary wanted this new interface feature and iterable support! It's changing everything for me. 😅

**Comment by freekmurze** [comment]
Awesome work as always! Thank you very much.

I'll already pull this in the v6 branch to polish it a bit if needed. Even though there should be no breaking changes, I'll create a new major version, just be on the safe side.

**Pull request merged**